martha/martha_v3.js

```diff
@@ -6,91 +6,85 @@ const url = require('url');
 const BAD_REQUEST_ERROR_CODE = 400;
 const SERVER_ERROR_CODE = 500;
 
+const BOND_LINK_NONE = null; // Used for servers that should NOT contact bond
+const BOND_LINK_DCF_FENCE = 'dcf-fence'; // The default when we don't recognize the server
+const BOND_LINK_FENCE = 'fence';
+const BOND_LINK_ANVIL = 'anvil';
```

Comment:
Why not use BondProviders from [here](https://github.com/broadinstitute/martha/blob/3a1acc3f9b68fcfe3c118d16ce87eba3822128af/common/bond.js#L6-L13)?

Reply:
Unlike the other endpoints, `martha_v3` reduced its dependencies and reimplements the common logic in `bond.js`. I'm happy to consider changing this if we discuss.